server/routes/juror-management/juror-management.controller.js

```diff
@@ -2,10 +2,32 @@
   'use strict';
 
   const _ = require('lodash');
-  const { getJurorStatus } = require('../../lib/mod-utils');
+  const { getJurorStatus, setPreviousWorkingDay } = require('../../lib/mod-utils');
   const dateFilter = require('../../components/filters').dateFilter;
   const { jurorsAttending, poolAttedanceAuditDAO } = require('../../objects/juror-attendance');
 
+  module.exports.isAttendanceConfirmed = async function(app, req, locCode, attendanceDate) {
+      return isAttendanceConfirmedByAttendances(await getAppearances(app,req, locCode, attendanceDate));
```

Comment:
we should just probably wrap this on a try-catch block to capture an error if it happens on `getAppearances`

Reply:
Updated :)